core/src/main/scala/org/apache/spark/scheduler/SparkListener.scala

```diff
@@ -291,6 +294,11 @@ private[spark] trait SparkListenerInterface {
   def onBlockUpdated(blockUpdated: SparkListenerBlockUpdated): Unit
 
   /**
+   * Called when a speculative task is submitted
+   */
+  def onSpeculativeTaskSubmitted(speculativeTask: SparkListenerSpeculativeTaskSubmitted): Unit
```

Comment:
For normal task, we have `onTaskStart`, `onTaskEnd`, etc. but don't have `onTaskSubmitted`. Shall we make the name consistent for speculative task?